js/rank_states_utils.js

```diff
@@ -3,3 +3,142 @@ var svgStates = d3.select("#rank-states-interactive")
   .append("svg")
   .attr('viewBox', '0 0 1000 300')
   .attr('preserveAspectRatio', 'xMidYMid');
+
+var stateMap = svgStates.append('g')
+  .attr('id','ranked-states-map')
+  .attr('transform',"scale(0.4)");
+
+stateMap.append('g')
+  .attr('id','ranked-states-moved');
+
+stateMap.append('g')
+  .attr('id','ranked-states-dragable');
+  
+svgStates.append('g')
+  .attr('id','ranked-states-bars');
+  
+  
+var movedStates = ["AL","AK","AZ","AR","CA","CO","CT","DE","DC","FL","GA","HI","ID","IL","IN","IA","KS","KY","LA","ME","MD","MA","MI","MN","MS","MO","MT","NE","NV","NH","NJ","NM","NY","NC","ND","OH","OK","OR","PA","RI","SC","SD","TN","TX","UT","VT","VA","WA","WV","WI","WY","PR","VI"];
+
+var dragStates = ["ID","OK","MI"];
+
+// add states
+  svgStates.select('#ranked-states-moved')
+    .selectAll( 'use' )
+    .data(movedStates)
+    .enter()
+    .append('use')
+    .style('stroke-dasharray',"10, 10")
+    .classed('moved-state', true)
```

Comment:
calling these `moved-state` is a little confusing. It sounds like the ones that are going to be moved by the user. Maybe it should be `static-state` and `draggable-state`